packages/server/src/Server.ts

```diff
@@ -25,9 +25,14 @@ export const createServer = () => {
     const { roomName } = req.params;
     const { username } = req.body;
 
-    await deleteRoom(roomName, username);
-    res.status(200).send({});
-    await emitRoomUpdate(io);
+    try {
+      await deleteRoom(roomName, username);
+      res.status(200).send({});
+      await emitRoomUpdate(io);
+    } catch (e) {
```

Comment:
This is only for the `deleteRoom` right? If so it should only try/catch that.

Reply:
but should it emit anyway in case of error? maybe that's why it doens't work correctly ;-p

Reply:
If it should emit anyway (which is true), then it should be lower, outside of the try/catch. The `res.status` does not actually stop the execution of the rest, if I remember correctly.